src/config.py

```diff
@@ -33,3 +33,10 @@ def is_feature_flag_enabled(flag_name: str) -> bool:
 SGTM_FEATURE__AUTOMERGE_ENABLED = is_feature_flag_enabled(
     "SGTM_FEATURE__AUTOMERGE_ENABLED"
 )
+SGTM_FEATURE__FOLLOWUP_REVIEW_GITHUB_USERS = {
+    int(github_id)
```

Comment:
Looks like we don't load `id`s for authors (though we mock them out).  I'm going to switch this to take `user.login` instead - things will be a bit less unique since it's effectively a username which can collide for Github Apps, but I can't see that affecting people and it's not really a problem if it does collide.

Reply:
Ah - if you wanted to use ids you'd have to just add `id` to the author fragments like https://github.com/Asana/SGTM/blob/1d1c308f9754526c08ef86d8ce66958ecbc720b1/src/github/graphql/fragments/FullReview.py#L9

Reply:
I considered that, but decided that I'd have to add it to all the fragments to feel confident, and `login` seems good enough + is already enforced to exist.